pkg/config/setup/config_nix.go

```diff
@@ -60,18 +60,11 @@ const (
 // in time
 func osinit() {
 	// Agent binary
-	if installPathOverride, ok := os.LookupEnv("DD_TEST_INSTALL_PATH_OVERRIDE"); ok {
-		// Some tests use the InstallPath variable to fetch dependencies. These tests' executable.Folder()
-		// are relative to the test files, but dependencies are always put in the same place during the tests'
-		// executions. This environment variable allows us to override the install path during the tests.
-		InstallPath = installPathOverride
-	} else {
-		_here, err := executable.Folder() // {InstallPath}/bin/agent OR {InstallPath}/embedded/bin
-		if err != nil {
-			panic(fmt.Sprintf("Failed to get executable path: %v", err))
-		}
-		InstallPath = filepath.Join(_here, "..", "..")
+	_here, err := executable.Folder()
+	if err != nil {
+		panic(fmt.Sprintf("Failed to get executable path: %v", err))
 	}
+	InstallPath = getInstallPathFromExecutable(_here)
```

Comment:
out of curiosity (still have a lot to learn...): did we try leveraging `runtime.Caller(0)`?
`

Reply:
Merci for enlightening me, @BaptisteFoy!